base/server/python/pki/server/deployment/__init__.py

```diff
@@ -1392,13 +1392,19 @@ def import_master_config(self, subsystem):
 
             # check CSR in CS.cfg
             param = '%s.%s.certreq' % (subsystem.name, tag)
-            csr = subsystem.config.get(param)
 
-            if csr:
+            if tag == 'signing':  # for CA and OCSP
+                tag = subsystem.name + '_signing'
+
+            # store nickname and tokenname
+            if not ('pki_%s_csr_path' % tag) in self.mdict.keys():
+                continue
+
+            if self.mdict['pki_%s_csr_path' % tag]:
                 # CSR already exists
                 continue
```

Comment:
> the first `if` statement is probably never going to evaluate to `True`

Actually, I have added this `if` because I was getting `KeyError` exception in the TPS and TKS clone for the `subsystem` cert. This key is not defined for these subsystems.

Reply:
Yeah, you're right, it's not defined consistently in `default.cfg`.